dockerbuild/Dockerfile-backend

```diff
@@ -1,5 +1,7 @@
 FROM openjdk:8
 
+ARG THRIFT_VERSION=0.10.0
```

Comment:
This is unused in two places where `0.10.0` is hard-coded (lines 70 & 75 in this diff). You should use it in those locations.

Reply:
done

Reply:
Whoops, lines 30-33 also refer to this version! You should add it there as well.

Reply:
ahh sorry didn't see that! Added it there as well.